client/scss/components/_divider.scss

```diff
@@ -13,6 +13,10 @@
   background-color: color('pumice');
 }
 
+.divider--charcoal {
```

Comment:
I guess all of these modifiers aren't necessary anymore – we could use e.g. `bg-charcoal`.